src/scripts/views/repCardView.js

```diff
@@ -9,36 +9,36 @@ const repCardHandler = {};
 
 // renders the results of rep response
 repCardHandler.repCards = function(results, compiledTemplate, $parent) {
-  results.forEach(function(rep) {
-    if (rep) {
-      const newRep = new MoC(rep);
-      newRep.format();
-      $parent.append(compiledTemplate(newRep));
-    }
-  });
+  // results.forEach(function(rep) {
```

Comment:
it would be good to search for these functions, and as long as they're no longer called anymore (which they shouldnt be) you can fully delete these